### PR TITLE
updating multicluster over gateways

### PR DESCRIPTION
**Pull request by rshriram**
related to https://github.com/istio/istio/pull/9776

Signed-off-by: Shriram Rajagopalan <shriramr@vmware.com>

### CODE REVIEW

File: content/docs/setup/kubernetes/multicluster-install/gateways/index.md [hunk 1]
```diff
@@ -25,254 +25,220 @@ Cross-cluster communication occurs over Istio Gateways of the respective cluster
...
-and setting a default destination rule to originate an mTLS connection
-from every pod.
+## Configure DNS
 
-For example, apply the following global configuration in each cluster:
+Service from remote clusters that are named as
+`<name>.<namespace>.global` will be resolved by the CoreDNS addon shipped with Istio. 
+In order to utilize this DNS, Kubernetes' DNS needs to be
+configured to point to CoreDNS as the DNS server for the `.global`
+dns domain. Create the following config map (or update an existing one):
```
Comment (linsun) [inline content/docs/setup/kubernetes/multicluster-install/gateways/index.md]: would be good to mention briefly what permission is needed to edit the kube-dns configmap.

Comment (rshriram) [reply content/docs/setup/kubernetes/multicluster-install/gateways/index.md]: I dunno what they are. thats why I do this as a user initiated step - which implies user either has permissions to do this or doesn't.

Comment (linsun) [reply content/docs/setup/kubernetes/multicluster-install/gateways/index.md]: i see, thx, we can add them when we know :)

File: content/docs/setup/kubernetes/multicluster-install/gateways/index.md [hunk 2]
```diff
@@ -25,254 +25,238 @@ Cross-cluster communication occurs over Istio Gateways of the respective cluster
...
-  host: "*.local"
-  trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
+  name: kube-dns
+  namespace: kube-system
+data:
+  stubDomains: |
+    {"global": ["<ClusterIP_of_istiocoredns.istio-system.svc.cluster.local>"]}
 {{< /text >}}
```
Comment (frankbu) [inline content/docs/setup/kubernetes/multicluster-install/gateways/index.md]: How about making this an executable command?
```
kubectl apply -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  name: kube-dns
  namespace: kube-system
data:
  stubDomains: |
    {"global": ["$(kubectl get svc -n istio-system istiocoredns -o jsonpath={.spec.clusterIP})"]}
EOF
```

**Pull request merged**